.github/workflows/parse-langtags.yml

```
@@ -0,0 +1,58 @@
+# This workflow will get the latest langtags.json file to generate iso639.txt
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Update iso639.txt with parse-lantags
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every 6 months
+    - cron: '0 0 1 */6 *'
```

Even though WS tags aren't updated all that often, we want to be able to pick up the changes rather quickly and know about them.  So I suggest running this once a week as a compromise.  That way we'll only wait up to a week to be notified when new changes are published.

```suggestion
    # Every week
    - cron: '0 5 * * 0'
```